exaudfclient/base/javacontainer/javacontainer_impl.cc

```diff
@@ -64,6 +58,21 @@ JavaVMImpl::JavaVMImpl(bool checkOnly, bool noJNI, SwigFactory& swigFactory)
     }
 }
 
+void JavaVMImpl::parseScriptOptions(JavaScriptOptions::ScriptOptionsParser & scriptOptionsParser, SwigFactory& swigFactory) {
+    JavaScriptOptions::Extractor extractor(scriptOptionsParser, swigFactory);
+
+    DBG_FUNC_CALL(cerr,extractor.extract(m_scriptCode));  // To be called before scripts are imported. Otherwise, the script classname from an imported script could be used
```

Comment:
Does the comment still makes sense, I don't see the import here, doesn't the extract handle this, too

Reply:
True. I moved the comment to the correct location